build.gradle

```diff
@@ -295,5 +295,14 @@ taskTree{
 
 defaultTasks 'ciBuild'
 
-preBuild.dependsOn includeDirBugFix
+apply from: "$rootDir/ktlint.gradle"
```

Comment:
We should maybe move this to the [`apply` section](https://github.com/zcash/zcash-android-wallet-sdk/blob/31ab31f1c040ece25e7175555c6f1c46e9dd1e01/build.gradle#L45-L62) of the file